app/app.go

```diff
@@ -536,7 +544,8 @@ func NewDesmosApp(
 	)
 
 	// Add wasm module route to the ibc router, then set and seal it
-	ibcRouter.AddRoute(wasm.ModuleName, wasm.NewIBCHandler(app.WasmKeeper, app.IBCKeeper.ChannelKeeper))
+	// TODO re-add this in a further PR to update wasmd
```

Comment:
Shouldn't it be re-added since this PR also includes the wasmd updation?

Reply:
@dadamu Added back, thanks!